weblate/utils/requirements.py

```diff
@@ -169,10 +167,75 @@ def get_versions():
     return result
 
 
+def get_version_number_db(database, version):
+    """
+        Returns version of database in format as (Major Version.Minor Version.Revision)
+        from an integer. To know more about database version , refer <https://www.psycopg.org/docs/connection.html>
+    """
+    if database is "POSTGRESQL":
```

Comment:
This code seems used for postgres only, maybe it doesn't need the database arg and condition?

Reply:
I have fixed it as well.

Reply:
I'll use the flake8 and `pre-commit run --all` from now on .